ateam_radio_bridge/test/launch_tests/radio_discovery_test.py

```diff
@@ -181,10 +181,10 @@ def test_1_telemetryMessage(self):
     def test_2_commandMessage(self):
         data = self.sock.recv(508)
         expected_data = bytearray(b"\x00") * 36
-        expected_data[0:4] = [92, 255, 234, 93]  # CRC
+        expected_data[0:4] = [57, 152, 86, 229]  # CRC
```

Comment:
Why did we change the CRC here?

Reply:
The CRC is calculated from the packet content. Since that's changed, the CRC had to change.